lib/model/QubitNote.php

```diff
@@ -22,6 +22,37 @@ class QubitNote extends BaseNote
     // Flag for updating search index on save or delete
     public $indexOnSave = true;
 
+    public function __get($name) {
+        $args = func_get_args();
+
+        $options = [];
+        if (1 < count($args)) {
+            $options = $args[1];
+        }
+
+        if ($name === 'type' && $options['sourceCulture'] === true) {
+            //echo 'note id: ' . $this->id;
+            //echo 'type id: ' . $this->typeId;
+            //echo 'source culture: ' . $this->sourceCulture;
+
+            //$criteria = new Criteria();
```

Comment:
This commented out block was supposed to achieve the same result as the query below but I'm getting the incorrect record, and the result I get is for a different typeId for some reason.